lib/classes/ConfigSchemaHandler/index.js

```diff
@@ -98,7 +98,7 @@ class ConfigSchemaHandler {
       this.relaxProviderSchema();
     }
 
-    const ajv = new Ajv({ allErrors: true, coerceTypes: 'array', verbose: true });
+    const ajv = new Ajv({ allErrors: true, coerceTypes: 'array', verbose: true, strict: false });
```

Comment:
Ideally we should enable _strict_ mode.

I take that it implies more updates to our schemas, and it's better to propagate this to other PR (?)

Reply:
@medikoo, that is correct. There is quite a lot to be done to enable strict validation wihtout failing test. My idea was to tackle those in a different PR if you agree.

Reply:
Absolutely, good move 👍